src/GraphQL.Client/Hash.cs

```diff
@@ -0,0 +1,44 @@
+using System.Buffers;
+using System.Diagnostics;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace GraphQL.Client.Http;
+
+internal static class Hash
+{
+    private static SHA256? _sha256;
+
+    internal static string Compute(string query)
+    {
+        var expected = Encoding.UTF8.GetByteCount(query);
```

Comment:
Might be better to use GetMaxByteCount here if you're trying to super-optimize this method.

Reply:
I'm not trying to super-optimize, I just copied code from the main repo.
`query { ping }` yields 14 for `Encoding.UTF8.GetByteCount` and 45 for `Encoding.UTF8.GetMaxByteCount`. I'm fine either way but would like to have the same methods in both repos.

Reply:
Up to you.  Fine as-is